src/components/Timer.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import '../styles/Timer.css'
+
+function Timer() {
+    const [currentValue, setCurrentValue] = useState(0);
+    const [intervalId, setIntervalId] = useState(false);
+
+    const resetTimer = () => {
+        clearInterval(intervalId);
+        setIntervalId(false);
+        setCurrentValue(0);
+    }
+
+    const startTimer = () => {
+        if (!intervalId) {
+            const startTime = Date.now();
+            const interval = setInterval(() => {
+                const value = (currentValue + Math.round((Date.now() - startTime) / 100) / 10);
```

Comment:
Consider `Math.floor()` instead of `Math.round()` so that the first second ticks after 1 second has elapsed instead of 0.5 seconds elapsed.

Reply:
Fixed in combination with moving the truncating to the format function